doc/classes/ViewportTexture.xml

```diff
@@ -6,7 +6,6 @@
 	<description>
 		Provides the content of a [Viewport] as a dynamic [Texture2D]. This can be used to mix controls, 2D game objects, and 3D game objects in the same scene.
 		To create a [ViewportTexture] in code, use the [method Viewport.get_texture] method on the target viewport.
-		[b]Note:[/b] When local to scene, this texture uses [method Resource.setup_local_to_scene] to set the proxy texture and flags in the local viewport. Local to scene [ViewportTexture]s will return incorrect data until the scene root is ready (see [signal Node.ready]).
```

Comment:
Ooh, oops. They added that extra note (that is actually useful to know) AFTER this original PR. Gotta modify accordingly, but I would not like to mention the `setup_local_to_scene` part because it's such useless information to the typical user (last time I checked ViewportTexture is also the only class that does anything of the sort)

Reply:
Take a gander again, with that little bit of doc polishing I'm accustomed to.

Reply:
`local_to_scene` is true by default in ViewportTextures and it's even unadvised to disable it. So "[ViewportTexture]s with [member Resource.local_to_scene] set to [code]true[/code]" applies to every ViewportTexture pretty much.

Reply:
Took me a while to think about this, then.